vars/abapEnvironmentPipelineStageIntegrationTests.groovy

```diff
@@ -17,8 +21,23 @@ void call(Map parameters = [:]) {
     def script = checkScript(this, parameters) ?: this
     def stageName = parameters.stageName?:env.STAGE_NAME
 
+    Map config = ConfigurationHelper.newInstance(this)
+         .loadStepDefaults()
+         .mixinGeneralConfig(script.commonPipelineEnvironment, GENERAL_CONFIG_KEYS)
+         .mixinStageConfig(script.commonPipelineEnvironment, stageName, STEP_CONFIG_KEYS)
+         .mixin(parameters, PARAMETER_KEYS)
+         .use()
+
     piperStageWrapper (script: script, stageName: stageName, stashContent: [], stageLocking: false) {
-        cloudFoundryCreateService script: parameters.script
+        try {
+            cloudFoundryCreateService script: parameters.script
+        } catch (Exception e) {
+            script.currentBuild.result = 'UNSTABLE'
```

Comment:
In principal, we need to "try catch" the function cloudFoundryCreateService. If we don't do this, the step cloudFoundryDeleteService will not be called in case of an error.

Still, the test failed, therefore the result is set to 'UNSTALBE'. Of course, 'ERROR' would also be an option. But, as this is a failing test (and not an error in the pipeline execution), I would prefer 'UNSTABLE'.